docker/minimal-compose.yml

```diff
@@ -4,17 +4,7 @@ services:
     extends:
       file: ./gateway-compose.yml
       service: gateway
-    healthcheck:
```

Comment:
note: for context the `minimal-compose.yml` file originally had a different configuration than the `localdev/docker-compose.yml` that was removed

the goal of overwriting these values was to give sufficient timeouts during the `test-container.sh` script without needing to have overly long health checks in real usage


Open to this removal and updating the defaults to be larger, i guess it asks the expectations of these files:
As long as we tell end users that the values aren't meant for production and just local dev + release I'd say we just make the defaults long enough to be sufficient for the test-container.sh case?

Reply:
yes .. I agree that these files should be sufficient for local usage only. But we do need to document some advice for production setup.

Reply:
@rdsarvar what's your suggested times? is current conf in minimal-compose suggested onces?

After removing overwrites in minimal-compose, how about these confs?

## local dev + release testing

gateway-compose | before | after
-- | -- | --
interval(s) | 10 | 5
timeout(s) | 5 | 5
retries | 3 | 60
start_period(s) | 10 | 20

postgres-backend-compose | before | after
-- | -- | --
interval(s) | 10 | 1
timeout(s) | 3 | 1
retries | 3 | 60
start_period(s) | | 10

## production setup for docs

  | gateway-compose | postgres-backend-compose
-- | -- | --
interval(s) | 10 | 10
timeout(s) | 3 | 3
retries | 3 | 3
start_period(s) | 10 | 10

Reply:
Those values seem reasonable to me

Reply:
@rdsarvar one more question. in healthcheck for gateway-compose.yml file does not have test defined so how about adding 
`test: ["CMD-SHELL", "/usr/lib/trino/bin/health-check || exit 1"]`